StrategicInsightGroup.md

```diff
@@ -28,3 +34,7 @@ Appendix A.  ERDDAP Strategic Insight Groups Members (as of July, 2023)
 * Bob Simons - Creator of ERDDAP (Retired)
 * Micah Wengren - NOAA/NOS/US IOOS
 * Dale Robinson - NESDIS/CoastWatch, UCS/CIMEAS
+
+<footer>
+    <p>This document is built using GitHub pages. To propose changes to the document, see the GitHub source file at <a href="https://github.com/ERDDAP/erddap.github.io/blob/main/StrategicInsightGroup.md">https://github.com/ERDDAP/erddap.github.io/blob/main/StrategicInsightGroup.md</a></p>
```

Comment:
Maybe use https://github.com/ERDDAP/erddap.github.io/edit/main/StrategicInsightGroup.md to send folks directly into editing mode?